### PR TITLE
Bug 1136918 - Get rid of the link to hg.m.o in the revision list and move it to a separate button

### PR DESCRIPTION
**Pull request by KWierso**
[Review on Reviewable](https://reviewable.io/reviews/mozilla/treeherder/678)

### CODE REVIEW

**Comment by edmorley** [comment]
I think the fa-external icon adds quite a bit of visual clutter, so maybe we should omit it?
I also think it's more intuitive for the SHA to actually be a link, even if that means someone can't double click on it directly.

With both approaches, I'm unable to double click on the left side of the SHA and have the behaviour of TBPL. With the second approach I'm able to double click on the right hand side, but there is trailing whitespace (which didn't occur with the TBPL approach).

I've resurected TBPL locally so I can steal the markup it uses for reference - will paste in the bug :-)

**Comment by tojonmz** [comment]
> With the second approach I'm able to double click on the right hand side, but there is trailing whitespace (which didn't occur with the TBPL approach).

I wonder if this might be a variant of reopened core bug [303597](https://bugzilla.mozilla.org/show_bug.cgi?id=303597) affecting Firefox. Chrome is fine with the initial markup. If this approach was still required, as a workaround if we consolidate the revision and closing tag into a single line, the trailing whitespace goes away on Firefox.

`class="revision-text" target="_blank">{{revision}}</a>`

**Comment by tojonmz** [comment]
From a graphic design/layout point of view, I would think we'd want to have the resultset Date/Author, and its grey underline shoved to the right an equal amount (`20px`) so that everything continues to line up? Otherwise we have this:

layout

Instead of this:
proposed

File: ui/css/treeherder.css [hunk 1]
```diff
@@ -397,13 +397,16 @@ th-watched-repo {
     overflow: auto;
 }
 
-.revision > a:first-child {
-    float: left;
+.revision > span > a {
```
Comment (edmorley) [inline ui/css/treeherder.css]: This can be `.revision-holder > a {` yeah?

**Comment by tojonmz** [comment]
If we wanted to keep the alignment, perhaps these changes would be nice?

result-set {
~~border-top: 1px solid lightgrey;~~~
}

and
result-set-bar {
padding: 2px 0px 0px **34px**; 
}

Producing this:

resultsetbarbottomlineremoval

Just my opinion but the layout looks nice and more decluttered without that grey hairline. And it still gives us the 20px space to the left of the revision for dbl-click.

**Comment by edmorley** [comment]
So I tried this locally, and even with the `ondblclick` I get one space before and after the SHA.

Let's remove the `ondblclick` for now - and iterate on that later - even without it, the behaviour is still better than it was before :-)

**Pull request merged**